app/assets/stylesheets/content/_grid.sass

```diff
@@ -1,7 +1,7 @@
 $grid--gap-width: 20px
 $grid--header-width: 20px
 
-$grid--widget-padding: 20px
+$grid--widget-padding: 11px 20px 20px 20px
```

Comment:
I'm not so happy with all those magic numbers. I know that it is often a necessary evil to align things correctly. But it only gets worse when you add the spacing correction. Maybe you can look again if you can draw some numbers straight and the distance can't be 10px here for example. However this is has a rather low priority.

Reply:
I just changed that to 20px on all sides. But that is actuallay a coincidence. After talking to Christoph we decided to give the title more space on top than on the other sides.